plugin/trino-delta-lake/src/main/java/io/trino/plugin/deltalake/DeltaLakePageSourceProvider.java

```diff
@@ -159,9 +159,9 @@ public ConnectorPageSource createPageSource(
 
         // We reach here when we could not prune the split using file level stats, table predicate
         // and the dynamic filter in the coordinator during split generation. The file level stats
-        // in DeltaLakeSplit#filePredicate could help to prune this split when a more selective dynamic filter
+        // in DeltaLakeSplit#statisticsPredicate could help to prune this split when a more selective dynamic filter
         // is available now, without having to access parquet file footer for row-group stats.
-        // We avoid sending DeltaLakeSplit#splitPredicate to workers by using table.getPredicate() here.
+        // We avoid sending DeltaLakeSplit#statisticsPredicate to workers using split.getStatisticsPredicate() here.
```

Comment:
it looks like it's being sent to workers https://github.com/trinodb/trino/blob/eb265655e41d042425b0c2b30cf58afc45756e65/plugin/trino-delta-lake/src/main/java/io/trino/plugin/deltalake/DeltaLakeSplit.java#L146-L149


```suggestion
```

cc @alexjo2144